src/Heist/Extra/Splices/Pandoc/CtxSpec.hs

```diff
@@ -0,0 +1,38 @@
+module Heist.Extra.Splices.Pandoc.CtxSpec where
+
+import Hedgehog
+import Heist.Extra.Splices.Pandoc.Ctx
+import Relude
+import Test.Hspec
+import Test.Hspec.Hedgehog
+import Text.Pandoc.Definition
+
+spec :: Spec
+spec = do
+  describe "determineLinkType" $ do
+    it "recognizes internal URLs" . hedgehog $ do
+      determineLinkType [Str "example", Space, Str "link"] [] "/some/example/path#id" === LinkType_Internal
```

Comment:
I'd like to see this test made more readable. Instead of passing a parsed AST, just pass the raw markdown, viz.:

```haskell
determineLinkType "[example link](/some/example/path#id)" === LinkType_Internal
```

Reply:
Ok. I didn't want to involve the parser in order to keep the test "local" at the cost of readability. I'll improve that when I find another bit of free time. (Currently I can manage to spend just like 1hr a week on coding for fun...)